to_do_list/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import AssignmentsList from "./Components/List";
+import Header from "./Components/Header";
+import DownButtons from "./Containers/buttons";
+import config from './config';
+import storageManager from './StorageManager';
+
+function App() {
+  const [assignments, setAssignments] = useState({});
```

Comment:
why did you make `assignments` an object instead of array?
would be much simpler + logical to use an array to store multiple objects.
(also it would be much easier to handle)